libevmasm/PeepholeOptimiser.cpp

```diff
@@ -350,6 +397,33 @@ struct DoubleJump: SimplePeepholeOptimizerMethod<DoubleJump>
 	}
 };
 
+// rjumpi(tag_1) rjump(tag_2) tag_1: -> iszero rjumpi(tag_2) tag_1:
```

Comment:
Instead of `iszero rjumpi` could use `rjumpv` here, or perhaps have another peephole step for converting those cases.